### ISSUE TITLE
UE 5.2 build warnings

### ISSUE THREAD

**Issue by x157** [fpwong/BlueprintAssistWiki#120]
I'm running on the latest UE 5.2 from UDN.  While building the plugin I'm getting these warnings, which are only warnings, but posting here as an FYI.

```
C:\Users\xist\AppData\Local\Temp\a27d980e-cbb2-41a4-88ab-770516235714\HostProject\Plugins\BlueprintAssist\Source\BlueprintAssist\BlueprintAssistWidgets\BlueprintAssistGraphOverlay.cpp(286): warning C4996: 'FGeometry::ToPaintGeometry': Use FPaintGeometry ToPaintGeometry(const FVector2f& InLocalSize, const FSlateLayoutTransform& InLayoutTransform) instead. Please update your code to the new API before upgrading to the next release, otherwise your project will no longer compile.
```

```
C:\Users\xist\AppData\Local\Temp\a27d980e-cbb2-41a4-88ab-770516235714\HostProject\Plugins\BlueprintAssist\Source\BlueprintAssist\BlueprintAssistWidgets\BABlueprintActionMenu.cpp(220): warning C4996: 'FBlueprintActionMenuBuilder::FBlueprintActionMenuBuilder': The action filter now stores a reference to the authoritative editor context. Please use the default constructor instead. Please update your code to the new API before upgrading to the next release, otherwise your project will no longer compile.
```

**Comment by fpwong** [comment]
Hi Xist, thanks for letting me know. I'll fix this in the next update.

**Comment by fpwong** [comment]
Fixed in 4.1.7

**Comment by x157** [comment]
Confirmed fixed in 5.2, thanks @fpwong!
